WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift

```diff
@@ -558,7 +558,25 @@ class MeViewController: UITableViewController {
     /// into a self-hosted site the ability to create a WordPress.com account.
     ///
     fileprivate func promptForLoginOrSignup() {
```

Comment:
I didn't bother creating a feature flag, because I presume not many users sign in from here. I'm happy to add one though if you think that's necessary.

Reply:
If we are sure we are releasing it in 25.4, I think it should be OK. I'd add a flag and keep it until the entire app is updated to use web.